src/Http/Curators/CreateNewsArticleRequestHandler.php

```diff
@@ -45,6 +47,26 @@ public function handle(ServerRequestInterface $request): ResponseInterface
         /** @var NewsArticle $newsArticle */
         $newsArticle = $requestBodyParser->parse($request)->getParsedBody();
 
+        // Do not include publisher to search for duplicates, because each publisher already has its own URL anyway.
+        $existingNewsArticles = $this->newsArticleRepository->search(
+            new NewsArticleSearch(
+                null,
+                $newsArticle->getAbout(),
+                $newsArticle->getUrl()->toString()
+            )
+        );
+
+        if (!$existingNewsArticles->isEmpty()) {
+            /** @var NewsArticle $first */
+            $first = $existingNewsArticles->getFirst();
+            $id = $first->getId()->toString();
+
+            throw ApiProblem::bodyInvalidDataWithDetail(
+                'A news article with the given url and about already exists. (' . $id . ') '
+                    . 'Do a GET /news-articles request with `url` and `about` parameters to find it programmatically.'
```

Comment:
Is it not possible to get the news article with the `id`?

Reply:
Yes but this is to explain how a client can check themselves if an article with their `url` and `about` already exists _before_ doing a `POST` request. But it may not be that clear then 😄

Reply:
Oh! I understand now 🙂

Reply:
I'm going to leave it like this even if it's a bit confusing because it's super minor I think and updating it would require changes to this + unit tests + acceptance tests.